src/backend/routers/paras.ts

```diff
@@ -18,30 +17,61 @@ export const paraProcedures = {
       return result;
     }),
 
-  getAllParas: procedure.query(async (req) => {
-    const result = await req.ctx.db.selectFrom("user").selectAll().execute();
+  getMyParas: protectedProcedure.query(async (req) => {
+    const { userId } = req.ctx.auth;
+
+    const result = await req.ctx.db
+      .selectFrom("user")
+      .innerJoin("cm_to_para", "user.user_id", "cm_to_para.para_id")
+      .where("cm_to_para.case_manager_id", "=", userId)
+      .selectAll()
+      .execute();
 
     return result;
   }),
 
-  createPara: procedure
+  createParaAndAssignCaseManager: protectedProcedure
```

Comment:
In general, "and" should almost never be part of a function name - it's a code smell - it suggests that the function is doing too much. Consider structuring this as two functions - `createPara` and `assignParaToCaseManager`. This way, the two functions can be called together or separately as needed (we will need to call them separately if the CM is adding a para that already exists).

You can make `createPara` idempotent, where successive calls of the function for the same args are no-ops.